3_extracting_features/DP_files/NF1_annotations.csv

```diff
@@ -0,0 +1,9 @@
+Plate,Well,Gene Identifier,Gene Symbol,Genotype,Channels

```

Comment:
This file feels too important to belong in this folder. Should it belong somewhere else? Like some specified `metadata` folder?

Reply:
@gwaybio It is pretty important and it is good that you can specify where it comes from in the compile_DP_project.ipynb. Since in Cell Health, it was contained in the 0.download_data module so do you want me to add it to that module?

Reply:
yeah, I think that makes sense. I think it should be in a folder called `metadata` somewhere. This is probably in the `0.data-download` module.

In the README, you also should provide details on data provenance for this file. (i.e. how the file was created... did you create the file by hand? If so, this needs specification!)

Reply:
I will add a metadata file in this module until further notice. I want to make a plan for how to do this at one of our meetings to move this to 0.download-data since I don't want to mess anything up (even though there are no mistakes in Github :laughing:)